Source/Core/Core/IOS/VersionInfo.h

```diff
@@ -37,5 +37,30 @@ struct MemoryValues
 };
 
 const std::array<MemoryValues, 41>& GetMemoryValues();
+
+enum Feature
```

Comment:
For example, [std::launch](http://en.cppreference.com/w/cpp/thread/launch) is an enum class that satsfies the [BitmaskType](http://en.cppreference.com/w/cpp/concept/BitmaskType) concept.